packages/cli-kit/src/ui.ts

```diff
@@ -172,12 +172,23 @@ export async function terminateBlockingPortProcessPrompt(port: number, stepDescr
 }
 
 export const keypress = async () => {
-  process.stdin.setRawMode(true)
-  process.stdin.resume()
-  return new Promise<void>((resolve) =>
-    process.stdin.once('data', () => {
+  return new Promise((resolve, reject) => {
+    const handler = (buffer: Buffer) => {
+      process.stdin.removeListener('data', handler)
```

Comment:
❓ is this necessary? reading the code the listener is installed only once, so in theory if we are inside the callback there is no need to remove the listener, or am I reading this wrong?